ood-portal-generator/templates/ood-portal.conf.erb

```diff
@@ -155,6 +155,14 @@ Listen <%= addr_port %>
   #
   SetEnv OOD_PUN_STAGE_CMD "<%= @pun_stage_cmd %>"
 
+  <%- if @pun_pre_hook_exports -%>
+  # Environment variables to export to the PUN pre hook.
+  SetEnv OOD_PUN_PRE_HOOK_EXPORTS "<%= @pun_pre_hook_exports %>"
+  <%- if @pun_pre_hook_exports -%>
```

Comment:
I think this is wrong variable for this if condition, should be `@pun_pre_hook_root_cmd`

Reply:
:+1: Yep, fixed.